sched/mqueue/mq_rcvinternal.c

```diff
@@ -156,139 +147,90 @@ int nxmq_wait_receive(FAR struct mqueue_inode_s *msgq,
     }
 #endif
 
+  if (abstime)
```

Comment:
Do we need to check `abstime->tv_sec` and `abstime->tv_nsec` are non zero or non-NUL `abstime` ensures that non zero timeout is passed here?

Reply:
already check in file_mq_timedsend_internal() and file_mq_timedreceive_internal()